docs/streams/kafka_on_gdn/kafkajs-client.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 40
+title: KafkaJS Client
+---
+
+The KafkaJS client can be used to create Kafka producers, consumers. Full documentation [here](https://kafka.js.org/docs/getting-started).
+
+### Installation
+
+Install KafkaJS using:
```

Comment:
Do we need to tell people how to install or can you just link to the documentation?

Reply:
This step would be required as broker names are different in our case